src/modules/ffmpeg/producer/ffmpeg_producer.cpp

```diff
@@ -107,13 +107,13 @@ struct ffmpeg_producer : public core::frame_producer_base
 
     std::uint32_t frame_number() const override
     {
-        return static_cast<std::uint32_t>(producer_->time() * format_desc_.fps);
+        return static_cast<std::uint32_t>(producer_->time());
     }
 
     std::uint32_t nb_frames() const override
     {
         return producer_->loop() ? std::numeric_limits<std::uint32_t>::max()
-                                 : static_cast<std::uint32_t>(producer_->duration() * format_desc_.fps);
+                                 : static_cast<std::uint32_t>(producer_->duration());
```

Comment:
duration is AV_TIME_BASE time, not frame number?

Reply:
No, they are both frame numbers. We are treating them as such in various other places too, such as when outputing osc state or in the CALL handling